CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Version 2.12.0-beta1
  - [Enhancement to how QuickPulseTelemetryModule shares its ServiceEndpoint with QuickPulseTelemetryProcessor.](https://github.com/microsoft/ApplicationInsights-dotnet-server/pull/1266)
+ - [Adding a flag to DependencyTrackingTelemetryModule to enable/disable collection of SQL Command text](https://github.com/microsoft/ApplicationInsights-dotnet-server/pull/1285)
```

Comment:
we need to have this flag in Framework listenever as well (assuming Microsoft.Data. package approves your PR to add sql text)

https://github.com/microsoft/ApplicationInsights-dotnet-server/blob/develop/Src/DependencyCollector/Shared/Implementation/FrameworkSqlEventListener.cs

Reply:
Will do. The Microsoft.Data.SqlClient repo has merged the PR as of two days ago :)